helm-chart/templates/cr.yaml

```diff
@@ -0,0 +1,79 @@
+{{- if and .Values.createResource .Values.resource.tls.enabled .Values.resource.tls.useCertManager }}
```

Comment:
can we give a better name to this file? `cr` is very generic

Reply:
renamed to mongodbcommunity_cr_with_tls.yaml